migrations/Version202209081525261488_taoAdvancedSearch.php

```diff
@@ -25,21 +29,29 @@ public function getDescription(): string
         return 'Fix legacy configuration';
     }
 
+    /**
+     * @throws ServiceNotFoundException
+     * @throws InvalidServiceManagerException
+     * @throws common_Exception
+     */
```

Comment:
I do not really know why document it here if there is no client for this method that actually cares about any type of exception triggered.